x-pack/plugin/sql/connectors/tableau/connector/connectionBuilder.js

```diff
@@ -6,35 +6,7 @@
     } else{
         urlBuilder += "http://";
     }
-    urlBuilder += attr["server"] + ":" + attr["port"];
+    urlBuilder += attr["server"] + ":" + attr["port"] + "?";
```

Comment:
Is the question mark at the end always necessary?

Reply:
Yes: the "user" is a mandatory config option (can't submit the form if not filled), so there will always be a property set; so adding the `?` is safe.